pkg/plugin/validator.go

```diff
@@ -119,24 +121,34 @@ func (v *Validator) getAccessToken(ctx context.Context, repoName string) (string
 
 // parseIssueInfoFromURL parses pluginGitHubIssue from Issue URL.
 func parseIssueInfoFromURL(issueURL string) (*pluginGitHubIssue, error) {
-	if match, _ := regexp.MatchString(issueURLPatternRegExp, issueURL); !match {
-		return nil, fmt.Errorf("invalid issue url, issueURL doesn't match pattern: %s", issueURLPatternRegExp)
-	}
-	u, err := url.Parse(issueURL)
+	r, err := regexp.Compile(issueURLPatternRegExp)
```

Comment:
You generally want to do this compilation once at variable initialization, typically using `regexp.MustCompile`.